src/chroma/distance.nim

```diff
@@ -0,0 +1,66 @@
+## Implementation of CIEDE2000 color difference formula
+## 
+## See: https://en.wikipedia.org/wiki/Color_difference#CIEDE2000
+## 
+## The implementation follows: http://www2.ece.rochester.edu/~gsharma/ciede2000/ciede2000noteCRNA.pdf
+
+import colortypes, transformations, math
+
+const twentyfiveToSeventh = (25^7).float32
+
+proc myAtan(x, y: float32): float32 =
+  if x == 0 and y == 0:
+    return 0
+  elif x >= 0:
+    arctan2(x, y).radToDeg
+  else:
+    arctan2(x, y).radToDeg + 360
+
+func deltaE00*(c1, c2: ColorLAB, k_L, k_C, k_H = 1.float32): float32 =
+  let
+    C1 = sqrt(c1.a^2 + c1.b^2)
+    C2 = sqrt(c2.a^2 + c2.b^2)
+    CM = 0.5*(C1 + C2)
```

Comment:
`0.5*(`
I think there are some formatting things that nimpretty will fix.

Reply:
it was not fixed by nimpretty but I did it manually (I assume you imply spacing the '*')